### ISSUE TITLE
I can't see that ifttt trigger

### ISSUE THREAD

**Issue by aguiducci** [alexa/alexa-avs-sample-app#836]
My AlexaPi answer to all question about weather, time, news, general knowledge... but can't see any IFTTT trigger; each time she answers "I can't see that ifttt trigger" even if I created and reacreated it many times, tried to change phrase and disconnect and reconnect my amazon account to IFTTT.

Even stranger is that inside my smart home Alexa app I can see the history of all my conversation with Alexa, and how she listened my phrases...but question with "Alexa trigger..." are not even logged, like she neither heard something.

Do someone  already managed something like this?
Please give me some suggestions... thank you!!!

**Comment by lijessi** [comment]
Hi, have you tried linking your Alexa device to IFTTT on the IFTTT website? Could you please provide more details on things that you are trying to trigger? Thanks.

**Comment by aguiducci** [comment]
yes, my amazon alexa account is linked in my IFFT website. I'he also already tried to disconnect andconnect it again.
![screenshot_1](https://user-images.githubusercontent.com/27893908/27700876-534694fa-5cff-11e7-8a4a-db2554604786.png)

I'm trying to turn on device using a "Say a specific phrase" trigger:
![screenshot_2](https://user-images.githubusercontent.com/27893908/27700977-b12786f6-5cff-11e7-9296-6b61668119f7.png)

sending the data to my Adafruit IO MQTT broker:
![screenshot_3](https://user-images.githubusercontent.com/27893908/27700995-c3e7abcc-5cff-11e7-890c-6ba9ee51d65e.png)

Obviusly my trigger is setted to ON. But Alexa doesn't recognize it even if i change the phrase. Also triyed to simply send a gmail email to mail instead of using Adafruit IO

**Comment by lijessi** [comment]
Hi, thanks for your reply. Is the account you link with your AlexaPi the same as the one with your IFTTT account? And is it the US account?

**Comment by aguiducci** [comment]
Hi,

Yes is the same but is not US, is mandatory to be US?omg

**Comment by lijessi** [comment]
I am sure it is launch in US and AVS support that. But I am not sure about whether it is supported by your region. Perhaps you should take a look at the regions ifttt launched. Thanks:)

**Comment by mradulan** [comment]
can you try using an existing trigger https://ifttt.com/applets/284243p-tell-alexa-to-email-you-your-shopping-list.

**Comment by niccolospa** [comment]
I'm having the exact same issue, I tried also this one: 
I'm from Italy, so when in IFTTT I'm connecting to amazon account, in the login webpage I got, I changed all that in URL shows as it_IT to en_EN (two occourrences). That prompts me the US login, after logging, it redirects to IFTTT page and says that is connected, but even if I set several applets, Alexa is still  saying that cannot se the IFTTT trigger...

**Comment by dejavu1987** [comment]
I am in Germany, I have the same issue.

**Comment by esfera73** [comment]
Hello, I'm from Spain and same issue here!
